roofit/roostats/inc/RooStats/ToyMCSampler.h

```diff
@@ -147,65 +146,65 @@ class ToyMCSampler: public TestStatSampler {
       }
 
 
-      // Set the Pdf, add to the the workspace if not already there
+      /// Set the Pdf, add to the the workspace if not already there
       virtual void SetParametersForTestStat(const RooArgSet& nullpoi) {
          fParametersForTestStat.reset( nullpoi.snapshot() );
       }
 
       virtual void SetPdf(RooAbsPdf& pdf) { fPdf = &pdf; ClearCache(); }
 
-      // How to randomize the prior. Set to NULL to deactivate randomization.
+      /// How to randomize the prior. Set to NULL to deactivate randomization.
```

Comment:
/// Set a prior pdf to randomize nuisance parameters. 
/// Set to NULL to de-activate parameter randomisation